### PR TITLE
Support setStatus endpoint on memsource

### PR DESCRIPTION
**Pull request by ryanannatuvasquez**
## Description
- Support `setStatus` endpoint on memsource
https://wiki.memsource.com/wiki/Job_API_v7#Set_Status

### CODE REVIEW

File: memsource/api.py [hunk 1]
```diff
@@ -365,6 +365,7 @@ class Job(BaseApi):
     """You can see the document http://wiki.memsource.com/wiki/Job_API_v7
     """
 
+    status_completed = "Completed"
```
Comment (jcrevits) [inline memsource/api.py]: why not have this in the `constants` file? 🤔

Comment (ryanannatuvasquez) [reply memsource/api.py]: oh you are correct this should be on constants file.

**Comment by ryanannatuvasquez** [comment]
@jcrevits PTAL

File: memsource/api.py [hunk 2]
```diff
@@ -576,6 +576,20 @@ def delete(self, job_part_id: int, purge: bool=False) -> None:
             'purge': purge
         })
 
+    def setStatus(self, job_part_id, status) -> None:
```
Comment (jcrevits) [inline memsource/api.py]: oh my bad, didn't notice that at first but you didn't specify the argument types in the function declaration 🙇
at the same time maybe change `:param status:` to indicate that it should be a constant and then LGTM! 👍

**Comment by ryanannatuvasquez** [comment]
@jcrevits changes applied https://github.com/gengo/memsource-wrap/pull/61/commits/990f0f48eecd8929b9af5f7fe3cbdd868c52a51c, Thankyou PTAL

**Pull request merged**